src/login/LoginPage.jsx

```diff
@@ -229,6 +229,7 @@ const LoginPage = (props) => {
             name="password"
             value={formFields.password}
             autoComplete="off"
+            showScreenReaderText={false}
```

Comment:
For my understanding, why is this prop introduced?

Reply:
The sr-only message ("Password must contain at least 8 characters, at least one letter, and at least one number". ) on the password field was written considering the tooltip in mind that appears on the registration page. The tooltip is not accessible for screen readers and they need a way to see the password requirements. 

This message is however not needed on the login page.